src/Paket.Core/FrameworkHandling.fs

```diff
@@ -222,6 +225,7 @@ module FrameworkDetection =
                 | "dnx451" -> Some(DNX FrameworkVersion.V4_5_1)
                 | "dnxcore50" | "netplatform50" | "netcore50" | "aspnetcore50" | "aspnet50" | "dotnet" -> Some(DNXCore FrameworkVersion.V5_0)
                 | v when v.StartsWith "dotnet" -> Some(DNXCore FrameworkVersion.V5_0)
+                | v when v.StartsWith "netstandard" -> Some(DotNetStandard "1.5")
```

Comment:
Does this gloss over the differences between "netstandard1.0", "netstandard1.3" and "netstandard1.5" and does it mater?  thanks